simple-social-icons.php

```diff
@@ -58,6 +58,13 @@ class Simple_Social_Icons_Widget extends WP_Widget {
 	 */
 	protected $active_instances;
 
+	/**
+	 * Constrols custom css output.
```

Comment:
Looks like a typo?

Constrols → Controls

Reply:
What can I say about that? 🤦‍♂️

Reply:
Should be fixed by the last commit.